index.html

```diff
@@ -215,10 +215,10 @@ <h2>Securing JSON-LD Verifiable Credentials with JOSE</h2>
         </p>
         <p>[[rfc7515]] MAY be used to secure this media type.</p>
         <p>
-          The <code>typ</code> parameter MUST be <code>vc+ld+json+jwt</code>
+          The <code>typ</code> parameter SHOULD be <code>vc+ld+json+sd-jwt</code>
         </p>
         <p>
-          When present, the <code>cty</code> MUST be
+          When present, the <code>cty</code> SHOULD be
```

Comment:
Below is a snippet from the `application/vc+ld+json+sd-jwt` tab, as rendered by the respec examples plugin:
```
---------------- Decoded Protected Claimset ----------------
{
  "@context": [
    "https://www.w3.org/ns/credentials/v2",
    "https://www.w3.org/ns/credentials/examples/v2"
  ],
  "issuer": "https://example.edu/issuers/14",
  "issuanceDate": "2010-01-01T19:23:24Z",
  "credentialSubject": {
    "degree": {
      "name": "Bachelor of Science and Arts"
    },
    "id": "did:example:123"
  },
  "iss": "did:web:issuer.example",
  "iat": 1692916707,
  "exp": 1724539107,
  "cnf": {
    "jwk": {
      "kty": "EC",
      "crv": "P-384",
      "alg": "ES384",
      "x": "VCJxwsrpbSJwIk7LlGNeCCyH6SIe74Uc7YrWapZlaqWdsVfDOfeHWAjQQMLAETUL",
      "y": "pu2bGKexZLPyyc1ljPGg5Tj57tXSxsKvFK4vN2eGUngMGwpYNAtHjgpQ3pgLvUGI"
    }
  }
}
```

As you can see, it has the `iss`, `exp`, `iat`, ... claims. Is that intentional? 

I thought we had settled that the Decoded Protected Claimset should be the `application/vc+ld+json` only.

Reply:
is there guidance on how to interpret when `issuer` and `iss` values are different? like in the example above..?

Reply:
not in this PR, there are many open related issues.

One of the challenges addressing them has been no good way to talk about "controller documents" aka "did documents" 

After https://github.com/w3c/vc-jose-cose/pull/144

we should be able to address all the cases where "key discovery" leads to a verification key, and then we can address recommendations regarding `issuer`, and `proof` in the case they are present and disagree with `iss` and `kid`.

There is also a related PR open in the core data model that touches on this topic see: https://github.com/w3c/vc-data-model/pull/1259#discussion_r1310766445